bundle/manifests/tempo-operator-manager-config_v1_configmap.yaml

```diff
@@ -23,7 +23,7 @@ data:
     # after the manager stops then its usage might be unsafe.
     # leaderElectionReleaseOnCancel: true
     images:
-      tempo: docker.io/grafana/tempo:1.5.0
+      tempo: docker.io/grafana/tempo:2.0.0
```

Comment:
Do we need to update tempo-query then too?

Reply:
It will be good I think

Reply:
I won't do it, since the githash you need is not included in 2.0.0, is 2.0.0+